### DISCUSSION TITLE
Login Fail: Google reCAPTCHA verification failed, please try again later

### DISCUSSION THREAD

**Discussion by BilalMahmud12** [swellstores/community#181] [Help] [answered]
Hi There,
I tried to login many times but every time it shows google re-captcha failure, tried from different browsers - Chrome, Safari, Firefox but no luck 😔

Any one could help please?

![Screenshot 2023-03-18 at 17 24 48](https://user-images.githubusercontent.com/64046021/226112200-d9de253a-41fc-455b-b677-a87b0066472d.png)

**Comment by mustafaswell** [comment, marked as answer]
@BilalMahmud12 : looks like a related issue was reported and then eventually resolved by the looks of this [thread](https://github.com/orgs/swellstores/discussions/177).

Does the issue still exist?
